homeassistant/generated/config_flows.py

```diff
@@ -5,6 +5,341 @@
 
 # fmt: off
 
+FLOWS = [
```

Comment:
Please rebase on latest dev branch. This change is incorrect.

Reply:
Thanks - rebased